tests/phpunit/elementor/modules/dev-tools/test-deprecation.php

```diff
@@ -196,9 +196,59 @@ public function test_deprecated_function_soft() {
 		$this->assertEquals( [
 			'0.0.0',
 			'',
+			'Elementor',
 		], $settings['soft_notices']['test_deprecated_function_soft'] );
 	}
 
+
+	public function test_deprecated_function_as_admin() {
+		$this->act_as_admin();
+		$function_name = 'blablabla';
+		$this->setExpectedDeprecated( $function_name );
+		$this->run_function_deprecated_test( $function_name );
+	}
+
+	public function test_deprecated_argument_as_admin() {
+		$this->act_as_admin();
+		$argument_name = 'argumatura';
+		$this->setExpectedDeprecated( $argument_name );
+		$this->expectError();
+		$this->run_argument_deprecated_test( $argument_name );
+	}
+
+	public function test_deprecated_function_not_as_admin_does_not_show_message() {
+		$this->run_function_deprecated_test();
+		$this->assertTrue(true); // hack because phpunit demands assert
+	}
+
+	private function run_function_deprecated_test($function_name = 'blablabla'){
```

Comment:
Always prefer to use `test` as test data
```suggestion
	private function run_function_deprecated_test($function_name = 'blablabla'){
```